plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -3670,7 +3670,14 @@ function wc_logout_url( $redirect = '' ) {
  * @since 3.1.0
  */
 function wc_empty_cart_message() {
-	echo '<p class="cart-empty woocommerce-info">' . wp_kses_post( apply_filters( 'wc_empty_cart_message', __( 'Your cart is currently empty.', 'woocommerce' ) ) ) . '</p>';
+	echo '<div class="cart-empty">' .
+		wc_print_notice(
+			wp_kses_post( apply_filters( 'wc_empty_cart_message', __( 'Your cart is currently empty.', 'woocommerce' ) ) ),
```

Comment:
Does this need `esc_html__` or is it safe to skip because it's wrapped in `wp_kses_post`?

Reply:
Escaping should come last so I think its ok to let KSES deal with this one. The filter could also return HTML.